CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-## [0.3.0-dev] - 2021-01-28
+## [0.3.0] - 2021-02-23
 ### Added
-  * TBD
+  * Secured system container initial mounts (mount/remount/unmounts on these from within the container are now restricted). See [here](docs/user-guide/security.md#initial-mount-immutability) for details.
+  * Improved Sysbox systemd service unit files (dependencies, open-file limits).
+  * Improved logging by sysbox-mgr and sysbox-fs (json logging, more succint logs).
+  * Added support for systemd-managed cgroups v1 on the host (cgroups v2 still not supported).
+  * Added support for read-only Docker containers.
+  * Synced-up sysbox-runc to include the latest changes from the OCI runc.
+  * Added ground-work to support Sysbox on RedHat, Fedora, and CentOS (next step is creating a package manager for these).
+  * Fixed sysbox-mgr file descriptor leak (sysbox issue #195).
+  * Fixed problem with "docker --restart" on Sysbox containers (sysbox issue #184).
```

Comment:
* Fixed problem preventing kernel-headers from being properly imported within sys containers.
* Fixed inappropriate handling of mount instructions in chroot jail environments.

Reply:
>  Fixed problem preventing kernel-headers from being properly imported within sys containers.

Doesn't this fall under the bullet "Added ground-work to support Sysbox on RedHat, Fedora, and CentOS" ?

> Fixed inappropriate handling of mount instructions in chroot jail environments.

Don't recall this fix; was this specific to procfs/sysfs mounts in chroot jails inside a sys container?

Reply:
> Doesn't this fall under the bullet "Added ground-work to support Sysbox on RedHat, Fedora, and CentOS" ?

Nope. The original issue was seen in Ubuntu hosts deploying non-Ubuntu sys containers.

> Don't recall this fix; was this specific to procfs/sysfs mounts in chroot jails inside a sys container?

This one affected all mounts within a chroot jail inside a sys container. Sysbox-fs used to panic().

Reply:
Done.